projects/kit/src/lib/masks/number/utils/validate-pseudo-separators.ts

```diff
@@ -0,0 +1,17 @@
+import {DEFAULT_PSEUDO_SEPARATORS} from '../../../constants';
+
+export function validatePseudoSeparators({
```

Comment:
It is not obvious what kind these separators are.
Let's change in the following way:

```suggestion
export function validateDecimalPseudoSeparators({
```

(+ don't forget to change file name)